### PR TITLE
readme: Add myself to member list

### PR DESCRIPTION
**Pull request by kiennq**

### CODE REVIEW

**Comment by yyoncho** [comment]
Sorry for that omission.

File: README.md [hunk 1]
```diff
@@ -231,4 +231,17 @@ primary working on/responsible for.
       </div>
     </td>
   </tr>
+  <tr>
+    <td align="center">
+      <div>
+        <a href="https://github.com/kiennq">
+          <img src="https://github.com/kiennq.png" width="100px;" style="border-radius: 50%;" alt="danielmartin"/>
```
Comment (ericdallo) [inline README.md:238]: @kiennq I think you forgot to change the `alt`

Comment (yyoncho) [reply README.md:238]: I fixed it.

Comment (kiennq) [reply README.md:238]: Thanks :smile:

**Pull request merged**